### DISCUSSION TITLE
Option not to play next song

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#571] [Ideas]
I regularly use this app to do street dancing, and whenever I finish dancing to a song, I have to grab my phone quickly to stop the next song from playing (I need to think a bit before I decide what song to play next). For this very specific use case, it would be beneficial to have a setting that stops playback after a song has finished.

Are there other people interested in such a setting?

**Comment by naveensingh** [comment]
"Stop playback after current song" (repeat once) is one of the repeat options available. Isn't that sufficient for your use-case?

![image](https://github.com/FossifyOrg/Music-Player/assets/36371707/1a482dc3-da69-45ef-944b-2d08f6398412)

**Reply by ghost** [reply]
That option still plays the current song again (obviously). What I was thinking of was a way to stop all playback entirely after a song finishes.

**Reply by ghost** [reply]
Oh, damn, wait, I just found what you were mentioning! I never discovered that before! Thank you so much!
